extensions.json

```diff
@@ -944,7 +944,13 @@
     "repository": "https://github.com/paiqo/Databricks-VSCode"
   },
   "paragdiwan.gitpatch": {
-    "repository": "https://github.com/paragdiwan/vscode-git-patch"
+    "repository": "https://github.com/paragdiwan/vscode-git-patch",
+    "location": ".",
+    "custom": [
+      "git checkout a648bb3656d0c5c9e9c8cb281eb71d56fd0e9f6a",
+      "npm i",
+      "vsce package --out extension.vsxi"
```

Comment:
@manuth I think there is a typo with `.vsix`

Reply:
Oh - thanks for pointing this out 😄